### PR TITLE
DBZ-1303 Assembly without database drivers

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1303

### CODE REVIEW

File: debezium-connector-mysql/pom.xml [hunk 1]
```diff
@@ -390,6 +390,7 @@
               <mysql.port>4301</mysql.port>
               <mysql.gtid.port>4302</mysql.gtid.port>
               <mysql.gtid.replica.port>4303</mysql.gtid.replica.port>
+              <assembly.exclude.1>com.google.protobuf:protobuf-java:jar:*</assembly.exclude.1>
```
Comment (gunnarmorling) [inline debezium-connector-mysql/pom.xml]: Where is this one used?

Comment (jpechane) [reply debezium-connector-mysql/pom.xml]: Mysql driver pulls it in

Comment (jpechane) [reply debezium-connector-mysql/pom.xml]: @gunnarmorling It seems that protobuf is not necessary for JDBC-only usage so I am now excluding it at Maven dep level

Comment (gunnarmorling) [reply debezium-connector-mysql/pom.xml]: ok.

**Pull request merged**